### ISSUE TITLE
UpdateApp Could not load file or assembly Mono.Cecil

### ISSUE THREAD

**Issue by Slion** [Squirrel/Squirrel.Windows#1663]
**Squirrel version(s)**
Squirrel 2.0.0
WPF app

**Description**
UpdateApp throws following exception. 

```
System.IO.FileLoadException: Could not load file or assembly 'Mono.Cecil, Version=0.11.2.0, Culture=neutral, PublicKeyToken=50cebf1cceb9d05e' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
File name: 'Mono.Cecil, Version=0.11.2.0, Culture=neutral, PublicKeyToken=50cebf1cceb9d05e'
   at Squirrel.SquirrelAwareExecutableDetector.GetAssemblySquirrelAwareVersion(String executable)
   at Squirrel.SquirrelAwareExecutableDetector.<>c__DisplayClass1_0.<GetPESquirrelAwareVersion>b__0()
   at Squirrel.Utility.Retry[T](Func`1 block, Int32 retries)
   at Squirrel.SquirrelAwareExecutableDetector.GetPESquirrelAwareVersion(String executable)
   at Squirrel.SquirrelAwareExecutableDetector.<>c__DisplayClass0_0.<GetAllSquirrelAwareApps>b__2(String x)
   at System.Linq.Enumerable.WhereEnumerableIterator`1.MoveNext()
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at Squirrel.SquirrelAwareExecutableDetector.GetAllSquirrelAwareApps(String directory, Int32 minimumVersion)
   at Squirrel.UpdateManager.ApplyReleasesImpl.<invokePostInstall>d__10.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Squirrel.Utility.<LogIfThrows>d__42.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   at Squirrel.UpdateManager.ApplyReleasesImpl.<ApplyReleases>d__2.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Squirrel.UpdateManager.<ApplyReleases>d__10.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Squirrel.Utility.<LogIfThrows>d__43`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Squirrel.EasyModeMixin.<UpdateApp>d__0.MoveNext()
```

**Steps to recreate**
Run UpdateApp.

**Expected behavior**
Does not throw an exception.

**Actual behavior**
Throws an exception.

**Additional information**
Looks like this is a regression introduced in 2.0.0.
Looks like @anaisbetts fixed that 5 years ago in #329 but it came back.
Just tried 1.9.1 and it seems to be working fine.

**Comment by anaisbetts** [comment]
This is fixed in 2.0.1

**Comment by Slion** [comment]
@anaisbetts Thanks for the quick turnaround you are awesome. I thought you did not maintain that project any more.

Could you explain to us what you had to fix, for the record? Did something go wrong in the release process?

**Comment by Nextra** [comment]
@Slion Bug was discovered in #1654 and fixed by #1655.
